wdfn-server/waterdata/views.py

```diff
@@ -36,7 +36,7 @@ def provisional_data_statement():
 @app.route('/iv-data-availability-statement')
 def iv_data_availability():
     """Render the IV data availability statement page."""
-    return "Not yet implemented"
+    return render_template('iv-data-availability-statement.html')
```

Comment:
The template file should use underscores rather than dashes. That is the convention we have been using.

Reply:
argh! fixed.